sickbeard/helpers.py

```diff
@@ -114,65 +114,81 @@ def remove_non_release_groups(name, clean_proper=False):
     # Check your database for funky release_names and add them here, to improve failed handling, archiving, and history.
     # select release_name from tv_episodes WHERE LENGTH(release_name);
     # [eSc], [SSG], [GWC] are valid release groups for non-anime
+
+    # All lowercase and alphabetically sorted
+    groups = {
+        '-20-40',
```

Comment:
so it's n not working: u'SiMPLE@BluRG'

i"m running this to test: https://gist.github.com/fernandog/96b63b2e860ebce95657